packages/core/src/features/metrics/metrics-feature.ts

```diff
@@ -58,5 +73,6 @@ export const metricsFeature = getFeature({
 
     di.register(podDetailsMetricsInjectable);
     di.register(podDetailsContainerMetricsInjectable);
+    di.register(deploymentDetailsMetricsInjectable);
```

Comment:
No action required, this is also possible:
`di.register(clusterPieChartsClusterOverviewInjectable , clusterMetricsOverviewBlockInjectable, makeMargaritasInjectable, itPutsTheLotionInTheBasketInjectable);`